packages/libs/react-components/src/styles/stitches.config.ts

```diff
@@ -23,7 +24,16 @@ const sizes: Record<string, string> = {
   48: '12rem', // 192px
   56: '14rem', // 224px
   64: '16rem', // 256px
-};
+} as const;
+
+// eslint-disable-next-line @kadena-dev/typedef-var
+export const breakpoints = {
+  sm: `(min-width: ${640 / 16}rem)`,
+  md: `(min-width: ${768 / 16}rem)`,
+  lg: `(min-width: ${1024 / 16}rem)`,
+  xl: `(min-width: ${1280 / 16}rem)`,
+  '2xl': `(min-width: ${1536 / 16}rem)`,
+} as const;
```

Comment:
What is the reasoning for exporting these?

Reply:
for the typing. 
These are not in the theme object. so need them exported